3-JSS/package.json

```diff
@@ -14,7 +15,12 @@
     "babel-loader": "^8.2.3",
     "copy-webpack-plugin": "^10.2.4",
     "css-loader": "^6.6.0",
+    "cssjson": "^2.1.3",
+    "cssparser": "^0.9.4",
+    "fs": "^0.0.1-security",
     "html-webpack-plugin": "^5.5.0",
+    "jss": "^10.9.0",
+    "jss-preset-default": "^10.9.0",
```

Comment:
you're using the library in the actual production code, so it cannot be declared as `devDependency`. read more about it: https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file
🕺